include/fmt/core.h

```diff
@@ -1500,19 +1578,53 @@ class dynamic_format_arg_store
     if (internal::const_check(need_copy<T>::value))
       emplace_arg(dynamic_args_.push<stored_type<T>>(arg));
     else
-      emplace_arg(arg);
+      emplace_arg(static_cast<const internal::unwrap_reference_t<T>&>(arg));
```

Comment:
Same here. Then `unwrap_reference_t` can be removed.

Reply:
done